docs/se/openapi/openapi-ui.adoc

```diff
@@ -37,67 +37,75 @@ Also, make sure your project has the following dependency.
 
 The following example illustrates these steps, combining configuration with explicit programmatic settings.
 
 [source,java]
 .Create `OpenApiUi` and `OpenAPISupport` instances
 ----
 Config config = Config.create(); // <1>
-Config openApiConfig = config.get(OpenAPISupport.Builder.CONFIG_KEY)); // <2>
-
-OpenApiUi.Builder uiBuilder =
-    OpenApiUi.builder() // <3>
-        .webContext("/my-openapi-ui"); // <4>
-
-OpenAPISupport openApiSupport =
-    OpenAPISupport.builder() // <5>
-        .ui(uiBuilder)
-        .config(openApiConfig) // <6>
-        .build();
-
-
+Config.global(config);
+
+Config openApiConfig = Config.global().get("openapi"); // <2>
+WebServer server = WebServer.builder()
+        .config(config.get("server"))
+        .addFeature(OpenApiFeature.builder() // <3>
+                            .addService(OpenApiUi.builder() // <4>
+                                                .webContext("my-ui") // <5>
+                                                .config(openApiConfig.get("ui")) // <6>
+                                                .build()) // <7>
+                            .config(openApiConfig)
+                            .build()) // <8>
+        .routing(Main::routing)
+        .build()
+        .start();
 ----
-<1> Load the configuration.
-<2> Extract the `OpenAPISupport` configuration.
-<3> Create the `OpenApiUi.Builder` instance.
-<4> Explicitly set the web context where the UI should respond.
-<5> Create the `OpenAPISupport` instance using the `OpenApiUi.Builder` just created.
-You can refine the behavior of the `OpenAPISupport` object by invoking additional methods on its builder before invoking its `build` method.
-<6> Apply the `openapi` configuration to the `OpenAPISupport` builder. This also automatically applies any `openapi.ui` configuration to the UI.
-
-The order in which your code invokes the methods on `OpenApiUi.Builder` and `OpenAPISupport.Builder` determines the outcome.
-For instance, the example above sets the UI on the `OpenAPISupport.Builder` _before_ applying configuration.
+<1> Load and set global configuration.
+<2> For convenience extract the `openapi` config.
+<3> Begin setting up the `OpenApiFeature` builder.
+<4> Create the UI builder.
+<5> Set UI behavior programmatically.
+<6> Set additional UI behavior based on UI configuration.
+<7> Build the `OpenApiUi` object.
+<8> Build the `OpenApiFeature` object.
+
+The order in which your code invokes the methods on `OpenApiUi.Builder` and `OpenApiFeature.Builder` determines the outcome.
+For instance, the example above add the UI service to the `OpenApiFeature.Builder` _before_ applying configuration to the `OpenApiFeature.Builder`.
 If the configuration contains a setting for the UI's `web-context` value then the UI uses the configured value, not the programmatic value, because your code applied the configuration later.
```

Comment:
If the configuration contains a setting for the UI's `web-context` value, then the UI uses the configured value and not the programmatic value because your code will apply the configuration later.

Reply:
Fixed